package.json

```diff
@@ -62,13 +62,13 @@
     "react-dom": "16.13.1",
     "react-native": "0.63.2",
     "react-native-testing-library": "^2.1.0",
-    "react-native-vector-icons": "^6.6.0",
+    "react-native-vector-icons": "^7.0.0",
     "react-test-renderer": "^16.13.1",
     "rimraf": "^3.0.2",
     "typescript": "^3.9.5"
   },
   "peerDependencies": {
-    "react-native-vector-icons": ">6.6.0"
+    "react-native-vector-icons": ">7.0.0"
```

Comment:
Let's add `react` too.

Reply:
I guess we can, though that's a little outside the scope of this PR.